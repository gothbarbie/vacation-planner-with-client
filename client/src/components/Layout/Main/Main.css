.main {
  display: flex; 
  justify-content: center;
  height: 100%;
}

.main__inner {
  width: 100%;
  max-width: 1140px;
  height: 100%;
  display: flex;
  flex-direction: row;
  padding: .5rem;
}
