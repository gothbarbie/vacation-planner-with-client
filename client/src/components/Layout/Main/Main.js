// @flow

import * as React from 'react'

import './Main.css'

type Props = {
  children: React.ChildrenArray<any>,
}

const Main = ({ children }: Props) => (
  <main className="main">
    <div className="main__inner">{children}</div>
  </main>
)

export default Main
