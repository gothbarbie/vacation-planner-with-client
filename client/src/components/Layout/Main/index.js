// @flow

import React from 'react'

import './Main.css'

type props = {
  children: ReactElement,
}

export default ({ children } : props) => (
  <main className="main">
    <div className="main__inner">{children}</div>
  </main>
)
