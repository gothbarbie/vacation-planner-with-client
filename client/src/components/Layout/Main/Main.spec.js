import React from 'react'
import { shallow } from 'enzyme'

import Main from './Main'

describe('components/Layout/Main', () => {
  const main = shallow(<Main />)

  it('renders', () => {
    expect(main).toMatchSnapshot()
  })
})
