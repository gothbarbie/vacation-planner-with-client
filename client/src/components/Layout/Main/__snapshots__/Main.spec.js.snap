// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Layout/Main renders 1`] = `
<main
  className="main"
>
  <div
    className="main__inner"
  />
</main>
`;
